cmd/table-manager/main.go

```diff
@@ -52,7 +52,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	tableClient, err := storage.NewTableClient(lastConfig.Store, storageConfig)
+	tableClient, err := storage.NewTableClient(lastConfig.IndexType, storageConfig)
```

Comment:
Yeah, thats right.  This will work for now, as long as chunks and index are in the same place, or if chunks are somewhere that doesn't need managing (ie S3 or GCS).  I'll file a follow up issue to clean this up.